### PR TITLE
Update README about setup steps

### PR DESCRIPTION
**Pull request by AleksSenkou**
Here:
1) `bin/update` is responsible only for updating dependencies
2) left notes about what should be moved to `bin/setup`

### CODE REVIEW

**Comment by AleksSenkou** [comment]
[I as developer want to generate documentation for api automatically](https://trello.com/c/Xr8uNixQ/7-i-as-developer-want-to-generate-documentation-for-api-automatically)

File: README.md [hunk 1]
```diff
@@ -13,27 +13,28 @@ We use:
...
+WIP: `yarn setup` script will be implemented for faster development start
 
-  visit _about://inspect_ in Chrome and click *Open dedicated DevTools for Node* link
+1) You may want to install packages for both server and client apps, here you are:
 
-  put `debugger` into your code
+in root dir: `yarn update`
 
-  enjoy
+2) Create database using psql:
```
Comment (DDKatch) [inline README.md]: @AleksSenkou we can create db via `node_modules/.bin/sequelize db:create` 😉

File: bin/update [hunk 2]
```diff
@@ -9,8 +9,10 @@ cd client && yarn
 echo "Updating server dependencies:"
 cd ../server && yarn
 
-echo "Set up env variables:"
-cp ../server/.env.sample ../server/.env
```
Comment (DDKatch) [inline bin/update]: @AleksSenkou looks like it should be in the updated version of this file

File: README.md (hunk 1 above)
Comment (AleksSenkou) [reply README.md]: @DDKatch can we do `NODE_ENV=test node_modules/.bin/sequelize db:create` for testing db creation?

File: bin/update (hunk 2 above)
Comment (AleksSenkou) [reply bin/update]: @datarockets/cuties ideally we need to copy the whole file only for development start, after that, we need only to add new env variables from `.env.sample` to `.env`

Here is a prototype for ruby:
bin/setup: https://github.com/datarockets/bevital/blob/master/bin/setup#L23
bin/update: https://github.com/datarockets/bevital/blob/master/bin/update#L37

Thoughts about having something like that in the future?

File: README.md (hunk 1 above)
Comment (DDKatch) [reply README.md]: @AleksSenkou yep

File: bin/update (hunk 2 above)
Comment (DDKatch) [reply bin/update]: @AleksSenkou I agree with that approach

**Comment by AleksSenkou** [comment]
@DDKatch fixed

**Review by DDKatch — APPROVED** [review]
LGTM

**Pull request merged**